scheduler.go

```diff
@@ -157,6 +154,11 @@ func (s *Scheduler) scheduleNextRun(job *Job) {
 }
 
 func (s *Scheduler) durationToNextRun(t time.Time, job *Job) time.Duration {
+	// job can be scheduled with .StartAt() and already has a next run
+	if job.nextRun.After(s.time.Now(s.Location())) {
```

Comment:
```suggestion
	if job.nextRun.After(t) {
```

Being ```t``` our lastRun time (or now if the job never ran) maybe that'd be the better choice to avoid minimum milliseconds differences between each call